### PR TITLE
fix: correct non scalar default arguments as props

### PR DESCRIPTION
**Pull request by chambo-e**
First PR to prepare for upcoming eslint config update

### CODE REVIEW

File: packages/ui/src/components/LineChart/index.tsx [hunk 1]
```diff
@@ -30,22 +30,27 @@ type LineChartProps = {
   chartProps?: Partial<LineSvgProps>
 }
 
+const defaultMargin = { bottom: 50, left: 60, right: 25, top: 50 }
```
Comment (matthprost) [inline packages/ui/src/components/LineChart/index.tsx:33]: Can all constant be UPPERCASE? As we use to do everywhere else

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by vincentaudebert — APPROVED** [review]

File: packages/form/src/components/DateField/index.tsx [hunk 2]
```diff
@@ -101,11 +101,11 @@ export const DateField = ({
           input.onChange(newDate)
         }
       }}
-      onBlur={(e: FocusEvent<HTMLElement, Element>) => {
+      onBlur={(e: FocusEvent<HTMLElement>) => {
```
Comment (QuiiBz) [inline packages/form/src/components/DateField/index.tsx:104]: I believe we shouldn't have to type again the parameters

File: packages/ui/src/components/BarChart/index.tsx [hunk 3]
```diff
@@ -29,13 +29,18 @@ type BarChartProps = {
   chartProps?: Partial<BarSvgProps<BarDatum>>
 }
 
+const defaultMargin = { bottom: 50, left: 60, right: 25, top: 50 }
```
Comment (QuiiBz) [inline packages/ui/src/components/BarChart/index.tsx:32]: Can you put these constants in uppercase?

**Review by QuiiBz — CHANGES_REQUESTED** [review]

File: packages/form/src/components/DateField/index.tsx (hunk 2 above)
Comment (chambo-e) [reply packages/form/src/components/DateField/index.tsx:104]: @QuiiBz 😢 

Screenshot 2023-03-02 at 15 49 45

File: packages/ui/src/components/BarChart/index.tsx [hunk 4]
```diff
@@ -29,16 +29,21 @@ type BarChartProps = {
   chartProps?: Partial<BarSvgProps<BarDatum>>
 }
 
+const DEFAULT_MARGIN = { bottom: 50, left: 60, right: 25, top: 50 }
+const DEFAULT_DATA: BarDatum[] = []
+const DEFAULT_AXISFORMATTER = {}
+const DEFAULT_KEYS = ['value']
```
Comment (QuiiBz) [inline packages/ui/src/components/BarChart/index.tsx:35]: Could we use the type from the props?
```suggestion
const DEFAULT_MARGIN: BarChartProps['margin'] = { bottom: 50, left: 60, right: 25, top: 50 }
const DEFAULT_DATA: BarChartProps['data'] = []
const DEFAULT_AXISFORMATTER: : BarChartProps['axisFormatters'] = {}
const DEFAULT_KEYS: : BarChartProps['keys'] = ['value']
```

File: packages/ui/src/components/DateInput/index.tsx [hunk 5]
```diff
@@ -165,11 +165,14 @@ type DateInputProps = Pick<
   value?: Date | string
 }
 
+const DEFAULT_FORMAT = (value?: Date | string) =>
```
Comment (QuiiBz) [inline packages/ui/src/components/DateInput/index.tsx:168]: Same here
```suggestion
const DEFAULT_FORMAT: DateInputProps['format'] = (value) =>
```

**Review by matthprost — APPROVED** [review]

**Pull request merged**